src/main.c

```diff
@@ -3,7 +3,8 @@
 #include <zephyr/kernel.h>
 #include <zephyr/device.h>
 #include <zephyr/devicetree.h>
-#include <zephyr/drivers/gpio.h>
```

Comment:
Would you mind keeping this?  I know we don't need it but the original main.c has it for unknown reason. And I'd like to keep the main.c as intact as possible.